### PR TITLE
Header footer dev branch

### PR DESCRIPTION
**Pull request by gsuarez17**
- Adds Top Bar and Footer components + storybooks
- Adds Material Symbols in place of Material Icons in index.html 
- One spacing variable added to styles.scss

#### Content Manager

**Article**

List view:

- [ ] Publication Status is working
- [ ] Review Workflows assignees & filtering is working (EE)

Entry view:

- [ ] CKEditor field is working
- [ ] Assignees and Stages are working (EE)

**Page**

Entry view:

- [ ] Creating and publishing a page is working

#### Plugins

**Content-type Builder**

- [ ] Content-type Builder is working

**Media Library**

- [ ] Media Library is working (upload)
- [ ] Folders are working

#### Settings

- [ ] Marketplace is working
- [ ] API Tokens Interface
- [ ] Internationalization Interface
- [ ] Single Sign-On Settings
- [ ] Review Workflows
  - Show custom created Workflow
  - Drag N Drop is working
  - Add new workflow stage is possible
  - Colors drop down is working
- [ ] Webhooks
  - Review Workflows Stage Change option
- [ ] Audit Logs
  - List view
  - Single log
- [ ] Roles
  - Default Roles
  - Create new role
  - Show CRUD + Publish
    - Field level
    - Locale
- [ ] Users & Permissions
  - Roles 
  - Add New Role

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<link href="https://fonts.googleapis.com/css2?family=Inter:ital,opsz,wght@0,14..32,100..900;1,14..32,100..900&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```
Comment (acamachoaltimetrik) [inline .storybook/preview-head.html:2]: why is this needed ?

File: src/app/components/shared/footer/footer.component.scss [hunk 2]
```diff
@@ -1,43 +1,98 @@
...
+    .nav-container {
+        
+        justify-content: flex-start !important;
+        nav {
+            flex-direction: column;
+            align-items: start;
+            ul {
+                flex-direction: column;
+                
+                li {
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/footer/footer.component.scss:15]: can we use the mixin here ? `@include respond-to("mobile-mb")`

File: src/app/components/shared/footer/footer.component.scss [hunk 3]
```diff
@@ -1,43 +1,98 @@
 $divider-color: #414641;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/footer/footer.component.scss:1]: can we check if we have this color already in the `variables.scss`?

File: src/app/components/shared/footer/footer.component.scss [hunk 4]
```diff
@@ -1,43 +1,98 @@
...
+
+        }
+    }
+}
+
+
 .footer {
+    position: absolute;
+    bottom: 0;
     background: var(--color-grey-90, #141414);
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/footer/footer.component.scss:33]: what about using/move this color to the variables.scss?

File: src/app/components/shared/footer/footer.component.scss [hunk 5]
```diff
@@ -51,16 +106,15 @@ $divider-color: #414641;
...
         align-items: center;
         -webkit-box-orient: vertical;
-        -webkit-line-clamp: 1;
-        align-self: stretch;
+        max-width: 100%;
         small {
+            display: block;
             overflow: hidden;
+            white-space: nowrap;
             color: var(--color-fg-tertiary, #787D78);
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/footer/footer.component.scss:114]: same as above

File: src/app/components/shared/top-bar/top-bar.component.html [hunk 6]
```diff
@@ -1,17 +1,45 @@
 <header class="header">
-    <div class="logo">
-        <img src="../../../../assets/icons/top-bar/Optic_Logo_White.svg" 
-        alt="Optic Logo">
+    <div class="lhs-container">
+        <button *ngIf="isLoggedIn" mat-icon-button (click)="onMenu()" class="menu-toggle lh-button">
+            <span class="material-symbols-outlined">
+                menu
+            </span>
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/top-bar/top-bar.component.html:6]: are you sure this one did not work with `<mat-icon>menu<mat-icon>`?

File: .storybook/preview-head.html (hunk 1 above)
Comment (gsuarez17) [reply .storybook/preview-head.html]: This is to allow icons and the google font to show up in storybook, but I forgot to update the second link there to material symbols as I did in index.html so I had to make a new commit just now for that.

File: src/app/components/shared/top-bar/top-bar.component.scss [hunk 7]
```diff
@@ -1,8 +1,35 @@
-$primary-color: #5A28FF;
+$primary-color: var(--hc-primary-color, #5A28FF);
 $primary-color-selected: #320FA0;
 $seconday-color: #FFFFFF;
 $divider-color: #A587FF;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/top-bar/top-bar.component.scss:5]: move this to` varaibles.scss`

File: src/app/components/shared/top-bar/top-bar.component.scss [hunk 8]
```diff
@@ -1,8 +1,35 @@
...
 
+
+@media (max-width: 480px) {
+    .header {
+        .lhs-container {
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/top-bar/top-bar.component.scss:9]: lets use the media query mixin

File: src/app/components/shared/top-bar/top-bar.component.html (hunk 6 above)
Comment (gsuarez17) [reply src/app/components/shared/top-bar/top-bar.component.html:6]: It did work with mat icon, I just switched them all to the material symbols syntax for continuity and since it is the new version that is supported

File: src/app/components/shared/top-bar/top-bar.component.scss (hunk 7 above)
Comment (gsuarez17) [reply src/app/components/shared/top-bar/top-bar.component.scss:5]: Should I copy/paste that line , entirely into variables.scss so that we can use it as $primary-color throughout the project or should I just delete it and use the var(_variableName_, #XXXXXX) format everywhere instead?

Comment (gsuarez17) [reply src/app/components/shared/top-bar/top-bar.component.scss:5]: It seems like it may be redundant to copy that line over
